### PR TITLE
Add favico

### PR DESCRIPTION
**Pull request by okom3pom**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Add favico
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no

### CODE REVIEW

**Review by NeOMakinG — APPROVED** [review]
I've nothing against it but is it the only thing needed? Maybe we could use Realfavicongenerator and get the full icon package for every devices, manifest in order to configure every color cases1

File: layouts/partials/head.html [hunk 1]
```diff
@@ -18,8 +18,7 @@
 {{ hugo.Generator }}
 
 <!--Favicon-->
-<link rel="shortcut icon" href="{{ site.Params.favicon | absURL }}" type="image/x-icon">
```
Comment (PierreRambaud) [inline layouts/partials/head.html:21]: You should keep this one for Chrome compatibility on Android.
https://github.com/audreyfeldroy/favicon-cheat-sheet#optional-but-encouraged

**Comment by okom3pom** [comment]
Hello @NeOMakinG 

I have copied what is on the dev doc.
Yes you are right but https://realfavicongenerator.net/ needs a 250*250 image 
What I don't have

File: layouts/partials/head.html (hunk 1 above)
Comment (okom3pom) [reply layouts/partials/head.html:21]: I will use Realfavicongenerator

**Comment by NeOMakinG** [comment]
> Hello @NeOMakinG
> 
> I have copied what is on the dev doc. Yes you are right but https://realfavicongenerator.net/ needs a 250*250 image What I don't have

Waiting for a designer response, I'll keep you in touch

**Comment by prestascott** [comment]
Hello everyone,

@NeOMakinG Here is the favicon in a 260*260 size:
Favicon-260x260

I hope it's good for you.

**Comment by prestascott** [comment]
Here is the favicon with a better resolution:
Favicon-260x260 (1)

**Comment by NeOMakinG** [comment]
@okom3pom didn't you forget to upload every images and manifest?

**Comment by okom3pom** [comment]
All images and manifest : https://github.com/PrestaShop/open-source/pull/75

**Review by NeOMakinG — APPROVED** [review]

File: layouts/partials/favicon.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<link rel="apple-touch-icon" sizes="180x180" href="/images/favicon/apple-touch-icon.png">
+<link rel="icon" type="image/png" sizes="32x32" href="/images/favicon/favicon-32x32.png">
+<link rel="icon" type="image/png" sizes="16x16" href="/images/favicon/favicon-16x16.png">
+<link rel="manifest" href="/images/favicon/site.webmanifest">
+<link rel="mask-icon" href="/images/favicon/safari-pinned-tab.svg" color="#5bbad5">
+<link rel="shortcut icon" href="/images/favicon/favicon.ico">
+<meta name="msapplication-TileColor" content="#da532c">
```
Comment (eternoendless) [inline layouts/partials/favicon.html:7]: This is an orangey color, I don't think it's going to look well with the icon?
Screenshot 2022-01-18 at 12 48 38

File: layouts/partials/head.html [hunk 3]
```diff
@@ -18,8 +18,7 @@
 {{ hugo.Generator }}
 
 <!--Favicon-->
-<link rel="shortcut icon" href="{{ site.Params.favicon | absURL }}" type="image/x-icon">
-<link rel="icon" href="{{ site.Params.favicon | absURL }}" type="image/x-icon">
+{{ partial "favicon.html" . }}
```
Comment (eternoendless) [inline layouts/partials/head.html:21]: I think the best implementation would be to keep the two original lines in the partial within this project, then override it in the site repo by placing it in `/layouts/partials/favicon.html`

That way the theme doesn't refer to files which don't exist in the theme

**Review by eternoendless — CHANGES_REQUESTED** [review]

File: layouts/partials/favicon.html (hunk 2 above)
Comment (okom3pom) [reply layouts/partials/favicon.html:7]: Hello @eternoendless.

I think it's a job for the product team to choose the color :)

File: layouts/partials/head.html (hunk 3 above)
Comment (okom3pom) [reply layouts/partials/head.html:21]: Ok, I also need to move `head.html` I'll do this

File: layouts/partials/favicon.html (hunk 2 above)
Comment (NeOMakinG) [reply layouts/partials/favicon.html:7]: Let's not wait for the product team on such a little thing I think so, I would use white color probably

**Comment by okom3pom** [comment]
I close this PR the files are moved to the project

**Comment by okom3pom** [comment]
It's a job for the PM :) mais i can change it.

File: layouts/partials/favicon.html (hunk 2 above)
Comment (okom3pom) [reply layouts/partials/favicon.html:7]: Capture d’écran de 2022-01-19 09-11-39

With the actual color

**Pull request closed without merging**